src/components/LowerRightControls.tsx

```diff
@@ -26,6 +39,19 @@ export function LowerRightControls({
     <section className="fixed bottom-2 right-2 flex flex-col items-end gap-3 pointer-events-none">
       {children}
       <menu className="flex items-center justify-end gap-3 pointer-events-auto">
+        {IS_NIGHTLY_OR_DEBUG && (
+          <Popover className="relative">
+            <Popover.Button className="p-0">
+              <BillingRemaining
+                mode={BillingRemainingMode.ProgressBarFixed}
+                billingActor={billingActor}
+              />
```

Comment:
`fmt` is probably needed to commit this suggestion, but I think this item should have a `Tooltip` like all the other `LowerRightControls`

```suggestion
              />
              <Tooltip position="top" contentClassName="text-xs">
                Text-to-CAD credits
              </Tooltip>

```